ts/features/payments/barcode/components/PaymentNoticeListItem.tsx

```diff
@@ -35,10 +36,13 @@ const PaymentNoticeListItem = ({
     <ListItemTransaction
       title={paymentNoticeNumber}
       subtitle={organizationFiscalCode}
-      transactionAmount={amountString}
+      transaction={{
+        amount: amountString,
+        amountAccessibilityLabel:
+          getAccessibleAmountText(amountString) ?? amountString
```

Comment:
Since `amountString` returns an empty string as its fallback in the worst-case scenario, is the additional fallback operation truly necessary?

Reply:
The `getAccessibleAmountText` utility function could return an `undefined` in its definition, so it means that this fallback operation is necessary.